ts/src/pro/gate.ts

```diff
@@ -289,7 +289,7 @@ export default class gate extends gateRest {
         const marketId = market['id'];
         const url = this.getUrlByMarket (market);
         const messageType = this.getTypeByMarket (market);
-        const [ topic, query ] = this.handleOptionAndParams (params, 'watchTicker', 'method', 'tickers');
+        const [ topic, query ] = this.handleOptionAndParams (params, 'watchTicker', 'name', 'tickers');
```

Comment:
yeah and besides that we have this in options 
![image](https://github.com/ccxt/ccxt/assets/43336371/93224d49-7d49-404c-9e6e-70731da46416)